### PR TITLE
Fetch with registrar

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

File: src/ecallmgr_registrar.erl [hunk 1]
```diff
@@ -194,6 +196,24 @@ collect_reg_success_props(Key, {FSJObj, Acc}) ->
         Value -> {FSJObj, [{Key, Value} | Acc]}
     end.
 
+-spec lookup_endpoint(kz_term:ne_binary(), kz_term:ne_binary()) ->
```
Comment (icehess) [inline src/ecallmgr_registrar.erl]: `s/kz_term:ne_binary/binary/`

File: src/ecallmgr_registrar.erl [hunk 2]
```diff
@@ -117,6 +118,7 @@
                       ,to_user = <<"nouser">> :: kz_term:ne_binary() | '_'
                       ,user_agent :: kz_term:api_ne_binary() | '_'
                       ,username :: kz_term:api_ne_binary() | '_'
+                      ,endpoint_info :: kz_json:api_object() | '_'
```
Comment (icehess) [inline src/ecallmgr_registrar.erl]: It seems the value is `kz_json:object()`. Also there is no type `kz_json:api_object` (it is `kz_term:api_object`)

**Review by icehess — CHANGES_REQUESTED** [review]

File: src/ecallmgr_registrar.erl (hunk 2 above)
Comment (lazedo) [reply src/ecallmgr_registrar.erl]: `kz_term:api_object` as it is created as undefined

File: src/ecallmgr_registrar.erl (hunk 1 above)
Comment (lazedo) [reply src/ecallmgr_registrar.erl]: lol, spec was copied from 
```
-spec lookup_proxy_path(kz_term:ne_binary(), kz_term:ne_binary()) ->
          {'ok', kz_term:api_ne_binary(), kz_term:proplist()} |
          {'error', 'not_found'}.
```
which i think was copied from
```
-spec lookup_contact(kz_term:ne_binary(), kz_term:ne_binary()) ->
          {'ok', kz_term:ne_binary(), kz_term:proplist()} |
          {'error', 'not_found'}.
```

and they both handle the `<<>>`

**Pull request merged**